### PR TITLE
Module 1 capstone project

### PR DESCRIPTION
**Pull request by lengushuru**
**Changes description**

-removed footer in about section
-added link to Logo
-changed the background to match closely the design
-removed commented codes
-added missing section
-satisfied all other requests by reviewer

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      </div>
+      <div class="event1">
+        <img src="./images/event2.jpeg" alt="">
+        <div class="layer">
+          <h3>samburu tech week in Wamba</h3>
+          <p>2019</p>
+        </div>
+      </div>
+    </div>
+  </section>
```
Comment (Abenezer-Tilahun) [inline about.html:87]: - [ ] As stated the project requirement The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable. So Between the "Home" section and "Past event" section you just missed one section from the design guidelines "Logo" section, So kindly check and fix it.

Your About page Look likes this : 

Capture

The Design Guidelines About Page Look likes this : 

Capture

File: about.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      <img src="./images/redbull-logo-vector.svg" alt="">
+      <img src="./images/nokia-logo-vector-1.svg" alt="">
+      <img src="./images/intel-logo-vector-1.svg" alt="">
+      <img src="./images/safaricom-removebg-preview.png" alt="">
+    </div>
+  </section>
+  <footer>
+    <h2>SAM<span class="logo">BURU</span></h2> 
+    <p>samburu university tech week <br>all rights reserved </p>
+  </footer>
```
Comment (Abenezer-Tilahun) [inline about.html:102]: - [ ]  As stated the project requirement The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable. So on the desktop version there is no "Partner" section Just like the design Guidelines and adjust the background color of footer section just like the design : So Kindly Fix it.

The design Guidelines About page Look likes this : 

Capture

Your Project About Page Look this : 

Capture

File: index.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        <ul>
+          <li><i class='bx bxl-facebook'></i></li>
+          <i class='bx bxl-twitter'></i>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <nav class="nav-bar">
+        <h2>SAM<span class="logo">BURU</span></h2>
```
Comment (Abenezer-Tilahun) [inline index.html:43]: - [ ] As stated the project requirement for interaction for links  I recommend you to make The logo in the header links to the home page.

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @lengushuru   , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linters error
✔️ You used correct GitHub Flow.
✔️ Professional README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+          <li class="nav-item"><a href="#">SPONSOR</a></li>
+          <li class="nav-item5"><a href="#">CAMPAIGN</a></li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+  </header>
+  <section class="home">
+    <p class="welcome">"hello tech world"</p>
+    <h1 class="page-title"> SAMBURU UNIVERSITY TECHNOLOGY<br> WEEK</h1>
```
Comment (ha-manel) [inline index.html:57]: - [ ] As requested by the previous reviewer, kindly make the first header an `h1` tag and the second one `h2` 👍

File: README.md [hunk 5]
```diff
@@ -242,44 +170,30 @@ Feel free to check the [issues page](../../issues/).
...
+<!-- > Write a message to encourage readers to support your project -->
 
-If you like this project...
+If you like this project Give a ⭐️!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (ha-manel) [inline README.md:181]: - [ ] As requested by the previous reviewer, kindly add a line here that gives credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    <h2 class="footer-title">Partner</h2>
+    <div class="underline"></div>
+    <div class="sponsor-logo">
+      <img src="./images/samburu-removebg-preview.png" alt=''>
+      <img src="./images/redbull-logo-vector.svg" alt="">
+      <img src="./images/nokia-logo-vector-1.svg" alt="">
+      <img src="./images/intel-logo-vector-1.svg" alt="">
+      <img src="./images/safaricom-removebg-preview.png" alt="">
+    </div>
+  </section>
```
Comment (ha-manel) [inline about.html:103]: - [ ] As mentioned by the previous reviewer, in the about page, the partners section is hidden in desktop view, but shown in mobile, and the footer is dark in desktop view but it's light colored in mobile view, kindly make the necessary changes to satisfy this description 👍 

## Your design:
image
image

## Guidelines:
image
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @lengushuru 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (Bondok6) [reply README.md:181]: - [ ] Your README is well documented 💯 But As the previous reviewers mentioned and according to the requirements, Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

> Requirement ⬇
image

File: about.html (hunk 6 above)
Comment (Bondok6) [reply about.html:103]: - [ ] As the previous reviewers mentioned, and according to the original design. ⬇
- About page should contain only dark `Footer` section in `Desktop version`, so kindly hide the partner section on desktop version and add dark footer section.
- About page should contain both the `Partner` section and white `Footer` section in `Mobile version`, so Kindly add white footer section in mobile version.

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @lengushuru 🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lengushuru** [comment]
required changes on the READMe file were already in the master README

**Review by bimbolabuari — COMMENTED** [review]
Hello @lengushuru  👋,

_This is an additional review due to the second opinion request._

- Kudos for working on your first capstone project ✊  
   - However, the previous CR was right about the two requested changes 
        - In the project requirement, we are asked to give **appropriate** credit to the author 
           which is required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/), Kindly add this to the 
            acknowledgement section of your README file explicitly stating [Cindy Shin in 
            Behance](https://www.behance.net/adagio07) the design author.
   

        - It is required that we follow the design guidelines, including colours, typography and layout (composition and space between elements).
       
Screenshot 2022-12-05 at 18 33 35
Screenshot 2022-12-05 at 18 42 13


Keep rocking 🚀!

**Pull request merged**